packages/smooth_app/lib/data_models/onboarding_loader.dart

```diff
@@ -40,8 +47,8 @@ class OnboardingLoader {
     }
   }
 
-  /// Actual download of all data.
-  Future<void> _downloadData() async =>
+  /// Actual download of all data, returns true if error thrown
```

Comment:
`returns true if error thrown`??? That may be right, if that's the case it's a good idea to add this comment, but that's rather counter-intuitive.
I'm old fashioned: returns `true` if ok, `false` is failed and throws exception if there's an exception.